src/types/types.ts

```diff
@@ -1724,7 +1724,10 @@ export interface EdgeAccount {
 
   // Verify existing credentials:
   readonly checkPassword: (password: string) => Promise<boolean>
-  readonly checkPin: (pin: string) => Promise<boolean>
+  readonly checkPin: (
+    pin: string,
+    opts?: { forDuressAccount?: boolean }
```

Comment:
I generally like to avoid the words `login` or `account` in core parameter names. Just `forDuress` would also be clear.

Reply:
Oh shoot. I agree, but we already use `forDuressAccount` in or API. We either:

1. Are inconsistent and use `forDuress` here.
2. Are consistent and use `forDuressAccount` here
3. Are consistent, use `forDuress` here, and add it everywhere else with `forDuressAccount` deprecated.

Reply:
We're going with #2